src/openads/infrastructure/position/positionCreatedObserver.js

```diff
@@ -0,0 +1,43 @@
+import {AD_AVAILABLE, AD_BAD_REQUEST, AD_ERROR, AD_NO_BID, AD_REQUEST_FAILURE} from '../connector/appnexus/event/events'
+
+const positionCreatedObserverFactory = appnexusConnector => appNexusConsumersRepository => ({payload, dispatcher}) =>
```

Comment:
Should not this observer be registered in the Container?

Reply:
You are completely right, I'm missing the build method and the registration to the Domain Event Bus!